title-updater.js

```
@@ -0,0 +1,103 @@
+        }
+
+        // Clean up the title (remove extra whitespace)
+        pageTitle = pageTitle.trim();
+
+        // Only update if we have a meaningful title and it doesn't already include our suffix
+        if (pageTitle && !pageTitle.includes('| Maxim Python SDK')) {
+            // Remove any existing suffixes that might be there
+            pageTitle = pageTitle.replace(/\s*\|.*$/, '');
+
+            // Add our custom suffix
+            document.title = pageTitle + ' | Maxim Python SDK';
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)